Makefile

```diff
@@ -135,7 +135,8 @@ dev-ms-deploy: dev kof-operator-docker-build ## Deploy `kof-mothership` helm cha
 
 	@$(YQ) eval -i '.kcm.kof.operator.image.repository = "kof-operator-controller"' dev/mothership-values.yaml
 	@$(call set_local_registry, "dev/mothership-values.yaml")
-	$(HELM) upgrade -i kof-mothership ./charts/kof-mothership -n kof --create-namespace -f dev/mothership-values.yaml
+	$(HELM) upgrade -i --wait --create-namespace -n kof kof-mothership ./charts/kof-mothership -f dev/mothership-values.yaml
+	kubectl rollout restart -n kof deployment/kof-mothership-kof-operator
```

Comment:
Let's try to avoid --wait flag and rollout restart. I understand it might be useful in some cases, but there are many cases, when a tiny changes in helm chart will cause a time waste.